src/browser_action/animate.css

```diff
@@ -0,0 +1,3272 @@
+@charset "UTF-8";
```

Comment:
NOTE: useful CSS animation library - https://github.com/daneden/animate.css. Using this for some status icon animations.